src/lib/components/Footer.svelte

```diff
@@ -0,0 +1,151 @@
+<script>
+  import Container from "$lib/components/Container.svelte"
+  import Grid from "$lib/components/Grid.svelte"
+  import Button from "$lib/components/Button.svelte"
+
+  import IOXIO from "$lib/images/ioxio-logo.svg?dataurl"
+  import mail from "$lib/images/mail-icon.svg"
+  import slack from "$lib/images/slack-icon.svg"
+  import arrow from "$lib/images/link-arrow-icon.svg?dataurl"
+
+  const SLACK_URL = "https://slack.ioxio.com"
+  const EMAIL_URL = "mailto:support@ioxio.com"
+</script>
+
+<footer>
+  <div class="top-section-wrapper">
+    <Container>
+      <Grid container class="footer-container">
+        <Grid sm={0} lg={1} />
+        <Grid sm={12} md={6}>
+          <div class="try-out-section">
+            <h3>Try out our technologies for free on</h3>
+            <div class="link">
+              <a href="https://sandbox.ioxio-dataspace.com"> IOXIO Dataspace&trade; Sandbox </a>
+              <img src={arrow} alt="arrow" />
```

Comment:
This arrow is not clickable which is rather unexpected

Reply:
![image](https://github.com/ioxiocom/ioxio-docs/assets/1296558/5be5c7a8-c34d-421e-af0a-58cac7e5bfb3)
And the alignment looks off?

Reply:
We don't want a screen reader to read "arrow", the `alt` doesn't add value here